internal/cmd/beta/volume/describe/describe.go

```diff
@@ -0,0 +1,156 @@
+			examples.NewExample(
+				`Show details of a volume with ID "xxx"`,
+				"$ stackit beta volume describe xxx",
+			),
+			examples.NewExample(
+				`Show details of a volume with ID "xxx" in JSON format`,
+				"$ stackit beta volume describe xxx --output-format json",
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(p, cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(p)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("read volume: %w", err)
+			}
+
+			return outputResult(p, model.OutputFormat, resp)
+		},
+	}
+	return cmd
+}
+
+func parseInput(p *print.Printer, cmd *cobra.Command, inputArgs []string) (*inputModel, error) {
+	volumeId := inputArgs[0]
+
+	globalFlags := globalflags.Parse(p, cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	model := inputModel{
+		GlobalFlagModel: globalFlags,
+		VolumeId:        volumeId,
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
+		if err != nil {
+			p.Debug(print.ErrorLevel, "convert model to string for debugging: %v", err)
+		} else {
+			p.Debug(print.DebugLevel, "parsed input values: %s", modelStr)
+		}
+	}
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *iaas.APIClient) iaas.ApiGetVolumeRequest {
+	return apiClient.GetVolume(ctx, model.ProjectId, model.VolumeId)
+}
+
+func outputResult(p *print.Printer, outputFormat string, volume *iaas.Volume) error {
+	switch outputFormat {
+	case print.JSONOutputFormat:
+		details, err := json.MarshalIndent(volume, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal volume: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(volume, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal volume: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	default:
+		serverId := ""
+		if volume.ServerId != nil {
+			serverId = *volume.ServerId
+		}
+
+		sourceId := ""
+		if volume.Source != nil {
+			sourceId = *volume.Source.Id
+		}
+
+		table := tables.NewTable()
+		table.AddRow("ID", *volume.Id)
+		table.AddSeparator()
+		table.AddRow("NAME", *volume.Name)
+		table.AddSeparator()
+		table.AddRow("STATE", *volume.Status)
+		table.AddSeparator()
+		table.AddRow("VOLUME SIZE (GB)", *volume.Size)
+		table.AddSeparator()
+		table.AddRow("PERFORMANCE CLASS", *volume.PerformanceClass)
+		table.AddSeparator()
+		table.AddRow("AVAILABILITY ZONE", *volume.AvailabilityZone)
+		table.AddSeparator()
+		table.AddRow("SOURCE", sourceId)
+		table.AddSeparator()
+		table.AddRow("SERVER", serverId)
+		table.AddSeparator()
```

Comment:
This will show these field even if they're empty, right? We can show conditionally

Reply:
Also we could show the size, if the field is present

Reply:
I think volume will always have a size even if we dont parse it in creation